libs/dh/metering-point/feature-identity-and-master-data/src/lib/identity/identity-text-field/dh-metering-point-identity-text-field-with-icon.component.html

```diff
@@ -0,0 +1,30 @@
+<!--
+@license
+Copyright 2020 Energinet DataHub A/S
+
+Licensed under the Apache License, Version 2.0 (the "License2");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<span
+  *ngIf="iconName"
+  ngClass="watt-space-inline-xs"
+  [ngStyle]="{ display: 'inline-block' }"
+>
+  <watt-icon
+    [name]="iconName"
+    [label]="iconName"
+    size="s"
+    aria-hidden
+  ></watt-icon>
+</span>
+
+<span class="watt-space-inline-m">{{ text }}</span>
```

Comment:
I can change it to flexbox 👍 

question: How do we ensure that gap, spacing, etc. follows the spacing from the design system?

Reply:
> This will also make it easier to set the spacing between the two elements using `gap`.

are we okay with using 'gap' with older versions of most browsers not supporting it ?

Reply:
The `gap` can be set to whatever the spacing between the elements should be. In Figma it's confusing whether the space between two elements is a specific value, or simply one empty space [using the spacebar]. My guess is that unless the space between elements is explicitly set to a value, then it can be treated as a simple spacebar (` `).

Reply:
> are we okay with using 'gap' with older versions of most browsers not supporting it ?

Yes. We don't have specific targets for supported browsers, so until we define them, our support is simply "latest versions of evergreen browsers".